students/hyerin/westagram/users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    name = models.CharField(max_length=50)
+
+    email = models.CharField(max_length=50)
+
+    password = models.CharField(max_length=50)
+
+    p_number = models.CharField(max_length=50)
```

Comment:
몇 가지 수정이 필요해 보여요~
1. class 내에 member 변수 선언시 빈 줄 삭제
2.`p_number` -> `phone_number` (변수명 줄여쓰지 않기, 파이썬 컨벤션 자료를 참고해주세요~)
3. 사용자 중복 저장을 피하기 위해서 unique 속성 추가
4. 이력 관리용 컬럼 추가 필요

```suggestion
    name = models.CharField(max_length=50)
    email = models.CharField(max_length=50)
    password = models.CharField(max_length=50)
    phone_number = models.CharField(max_length=50)
```